doc/100_references.rst

```diff
@@ -607,3 +624,140 @@ stored files) which files belong to what snapshot. When only these files
 are deleted, the particular snapshot vanished and all snapshots
 depending on data that has been added in the snapshot cannot be restored
 completely. Restic is not designed to detect this attack.
+
+Local Cache
+===========
+
+In order to speed up certain operations, restic manages a local cache of data.
+This document describes the data structures for the local cache with version 1.
+
+Versions
+--------
+
+The cache directory is selected according to the `XDG base dir specification
```

Comment:
This isn't quite right anymore, the XDG spec is only used on Linux (actually anything non-windows, non-darwin).

See also #1315